src/env/index.ts

```diff
@@ -0,0 +1,22 @@
+import * as envalid from 'envalid'
+import { singleton } from 'tsyringe'
+
+import { DEFAULT_CONFIG, DEFAULT_KEYS, defaultConfig, loadEnvAndOptions, PartialEnv } from './common.js'
+
+export type { PartialEnv } from './common.js'
+export { InitEnv } from './init.js'
+export { SmtpEnv } from './smtp.js'
+
+@singleton()
+export class Env<KS extends DEFAULT_KEYS = DEFAULT_KEYS> implements PartialEnv<KS> {
```

Comment:
what is the relation of InitEnv to Env?

Reply:
`InitEnv` is used by the init script not by the main application https://github.com/digicatapult/veritable-ui/blob/d089e533b3725df5261042223892fc50e6726f9f/src/init.ts

Reply:
yes I saw, so that means the initEnv is just exclusively for that and the actual Env is for runtime envars/ stuff after the init has happened?